### ISSUE TITLE
[Feature Request] Add Total number of books in library to the Home page [Books], Just like the Read and Unread does

### ISSUE THREAD

**Issue by jadehawk** [crocodilestick/Calibre-Web-Automated#153]
Not a problem Just a Feature Request.
I will like to have the Number of books in the library Always showing in the Home page (Which I refer to the "Books" page), The same way we can see the number of books we have or haven't read, this number is displayed nicely, just not for the Total of books.

Thank you.

![feature request](https://github.com/user-attachments/assets/5f426cdd-dd58-4217-a0e1-5b92fb75edb3)

**Comment by crocodilestick** [comment]
Good idea, added to the dev version already and prepared for next release 👍

**Comment by jadehawk** [comment]
> Good idea, added to the dev version already and prepared for next release 👍

You are the Best!!! Thank you.  I tried to figure how to do it,  but failed miserably. 

Thx again.

**Comment by jadehawk** [comment]
Thank you for Updating the DEV image in the Dockerhub (01-07-2025). The Changes look Perfect!
I have no idea how to generate a (nightly/daily) image so I was dependent on the "DEV" tag.
This is great!
Thank you once again.
![BookCount](https://github.com/user-attachments/assets/5b231e65-49a5-479b-926a-fabb85100a48)

**Comment by crocodilestick** [comment]
Added as of V3.0.1
